src/sensor/ping.h

```diff
@@ -409,29 +409,29 @@ class Ping : public Sensor
      * @brief Sensor variables
      */
 ///@{
-    QString _ascii_text;
-    QString _nack_msg;
-
-    uint8_t _srcId;
-    uint8_t _dstId;
-
-    uint8_t _device_type;
-    uint8_t _device_model;
-    uint16_t _fw_version_major;
-    uint16_t _fw_version_minor;
-
-    uint32_t _distance; // mm
-    uint8_t _confidence; // 0-100%
-    uint16_t _pulse_usec;
-    uint32_t _ping_number;
-    uint32_t _scan_start;
-    uint32_t _scan_length;
-    uint32_t _gain_index;
-    uint32_t _speed_of_sound;
-
-    uint16_t _processor_temperature;
-    uint16_t _pcb_temperature;
-    uint16_t _board_voltage;
+    QString _ascii_text = QString();
+    QString _nack_msg = QString();
+
+    uint8_t _srcId = 0;
+    uint8_t _dstId = 0;
+
+    uint8_t _device_type = 0;
+    uint8_t _device_model = 0;
+    uint16_t _fw_version_major = 0;
+    uint16_t _fw_version_minor = 0;
+
+    uint32_t _distance = 0; // mm
```

Comment:
Interesting, I would expect this to be a non-functional change at this point?

Reply:
This is only to avoid weird values in the graphic interface when not connected